Source/Urho3D/Navigation/CrowdAgent.h

```diff
@@ -163,6 +163,10 @@ class URHO3D_API CrowdAgent : public Component
     NavigationQuality navQuality_;
     /// Agent's Navigation Pushiness.
     NavigationPushiness navPushiness_;
+    /// Agent's previous target state used to check for state changes
```

Comment:
Not really important but please add period for all the Doxygen comment lines in the future for consistency sake.